### PR TITLE
Examples sorting order fixed, theming scripts added to repository

### PR DESCRIPTION
**Pull request by pseusys**
# Description

*Examples sorting order fixed, theming scripts added to repository.*

# Checklist

- [x] I have covered the code with tests
- [x] I have added comments to my code to help others understand it
- [x] I have updated the documentation to reflect the changes
- [x] I have performed a self-review of the changes

### CODE REVIEW

File: docs/source/utils/generate_notebook_links.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+from fnmatch import fnmatch
+from pathlib import Path
+from typing import List, Optional, Set
+
+
+def create_notebook_link(file: Path, notebook_path: Path):
```
Comment (avsakharov) [inline docs/source/utils/generate_notebook_links.py:6]: Should we have descriptions or comments in this file?

File: docs/source/utils/regenerate_apiref.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+from typing import List, Optional, Tuple, Dict
+
+
+def generate_doc_container(file: Path, includes: List[Path]):
```
Comment (avsakharov) [inline docs/source/utils/regenerate_apiref.py:5]: And the same question about this file.

**Review by avsakharov — APPROVED** [review]

**Review by kudep — APPROVED** [review]

**Review by kudep — CHANGES_REQUESTED** [review]
- [x] changes of notebooks
- [x] Alexander's changes request

**Comment by avsakharov** [comment]
For some reason, when I click on the Script section, the page scale decreases (from 100% to 75%).
image

**Review by avsakharov — APPROVED** [review]

**Review by kudep — APPROVED** [review]

**Review by kudep — CHANGES_REQUESTED** [review]
- [x] add into examples telegram

**Comment by avsakharov** [comment]
- [ ] script/core/message.py Write minimal comments

**Review by kudep — APPROVED** [review]

**Pull request merged**